style.css

```diff
@@ -66,11 +65,11 @@
 /* BODY SECTION */
 
 /* Body styling */
+
 body {
   width: 381px;
   height: 6251px;
   margin: auto;
```

Comment:
- [ ] Good job working on the mobile version 👍, however, make sure you use `font-family: 'Inter';` to stick to the design as it is required